website/models.py

```diff
@@ -155,6 +156,19 @@ def domain_name(self):
     def get_absolute_url(self):
         return "/domain/" + self.name
 
+    def get_twitter_account(self, slug):
+        if self.twitter:
+            return self.twitter
+
+        parsed_url = urlparse("http://" + slug)
+        validate = URLValidator(schemes=["https"])  # Only allow HTTPS URLs
+        try:
+            twitter_url = "https://twitter.com/%s" % (parsed_url.netloc.split(".")[-2:][0].title())
+            if validate(twitter_url):
+                self.twitter = twitter_url
```

Comment:
Shouldn't this return the result instead?

Reply:
done 👍